homeassistant/components/jvc_projector/binary_sensor.py

```diff
@@ -0,0 +1,48 @@
+"""Binary Sensor platform for JVC Projector integration."""
+
+from __future__ import annotations
+
+from jvcprojector import const
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import JvcProjectorDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import JvcProjectorEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the JVC Projector platform from a config entry."""
+    coordinator: JvcProjectorDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities([JvcBinarySensor(coordinator)])
+
+
+class JvcBinarySensor(JvcProjectorEntity, BinarySensorEntity):
+    """The entity class for JVC Projector integration."""
+
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
Hmm, I am leaning towards just removing this and adding the device class. This would make it `binary_sensor.jvc_somrthing_power` which is quite nice. name should only be set to None when its the main feature of the device. And I think remote is a more valid main feature than knowing if it has power. what do you think?

Reply:
@joostlek in reality I think the main feature is the Binary Sensor. The Remote is very dangerous, for example in my case I need to disable it as I do not want actions performed on a video projector remotely. Sending commands can screw up the setup and mess the configuration. The Power Status is needed before any communication to the porjector, needed in automations (i.e. open screen when on, dim lights when on, power on stereo when on etc...).
So I think is better to have the main feature the Binary Sensor and additional the Remote (and the sensor when I will add it :) )